webapp/src/components/link_tooltip/link_tooltip.test.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import Client from '@/client';
+
+import {LinkTooltip} from './link_tooltip';
+
+jest.mock('@/client', () => ({
+    getIssue: jest.fn(),
+    getPullRequest: jest.fn(),
+}));
+
+jest.mock('react-markdown', () => () => <div/>);
+
+describe('LinkTooltip', () => {
```

Comment:
Great job with the tests! They are a much appreciated addition

Could we also have a couple of tests that confirm that the "Opened by" link's href attribute changes accordingly based on the mapped props and component state?